gcp/src/main/java/org/apache/iceberg/gcp/GCPProperties.java

```diff
@@ -36,6 +37,19 @@ public class GCPProperties implements Serializable {
   public static final String GCS_CHANNEL_READ_CHUNK_SIZE = "gcs.channel.read.chunk-size-bytes";
   public static final String GCS_CHANNEL_WRITE_CHUNK_SIZE = "gcs.channel.write.chunk-size-bytes";
 
+  // For BigQuery BigLake Metastore.
+  // The endpoint of BigLake API.
+  // Optional, default to BigLakeCatalog.DEFAULT_BIGLAKE_SERVICE_ENDPOINT.
+  public static final String BIGLAKE_ENDPOINT = "biglake.endpoint";
```

Comment:
Why not use the standard catalog property `uri`?

Reply:
Changed to CatalogProperties.URI.